### ISSUE TITLE
is there a way to apply scheduled learning rate ? 

### ISSUE THREAD

**Issue by jediofgever** [eriklindernoren/PyTorch-YOLOv3#458]
I try to reduce the learning rate when after inital phase of learning rate, I tried to modify train.py accordingly but lr parameter in config file always overrides my change. Has anyone applied this ?

**Comment by Flova** [comment]
Lr scheduling has been added in a recent version.
